JS/Lesson22/tugba/style.css

```diff
@@ -0,0 +1,39 @@
+body {
+    font-family: Arial, sans-serif;
+}
+
+.accordion-container {
+    width: 60%;
+    max-width: 600px;
+}
+
+.accordion {
+    display: flex;
```

Comment:
Making the 'accordion' button a flex container is not really necessary, theres nothing inside it to allign.